src/index.html

```diff
@@ -48,7 +48,14 @@
       }
 
       window.onload = function() {
-        var url = "https://gentle-gorge-17630.herokuapp.com/api"
+        const env = Onfido.env
```

Comment:
I'm not sure if the webpack build script is transpiling javascript inside of an html. I would check that before using `const`.